app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -312,6 +316,11 @@ Result okResult(Object obj) {
         return ok((JsonNode)MAPPER.valueToTree(new ResourceList<T>(list)));
     }
     
+    Result okResult(ObjectWriter writer, Object object) throws JsonGenerationException, JsonMappingException, IOException {
+        String jsonString = writer.writeValueAsString(object);
+        return okResult(BridgeObjectMapper.get().readTree(jsonString));
+    }
```

Comment:
Would it be cleaner to just do this: https://stackoverflow.com/questions/22593471/setting-content-type-header-with-play-framework-java

Reply:
I found a way to create a Result object with the right content type that doesn't involve serializing/parsing again.